projects.md

```diff
@@ -15,3 +15,4 @@ Here are a list of projects that have or will be implementing ERC721A as part of
 - StarCatchers | Etherscan | [Twitter](https://twitter.com/StarcatchersNFT/)
 - [Lives of Asuna](https://livesofasuna.com/) | [Etherscan](https://etherscan.io/address/0xaf615b61448691fc3e4c61ae4f015d6e77b6cca8) | [Twitter](https://twitter.com/LivesOfAsuna)
 - [Rich Sad Cat](https://richsadcat.xyz/) | [Etherscan](https://etherscan.io/address/0x8dCCd0ac948d4dd262fBD0e679c3d54677F56B34) | [Twitter](https://twitter.com/richsadcatnft)
+- [MonkeyPoly](http://monkeypoly.com/ | [Etherscan](https://etherscan.io/address/0xc3d39585afC382527b9192C9032f443524b46457) | [Twitter](https://twitter.com/themonkeypoly)
```

Comment:
Looks like this was malformatted 😮

Reply:
Looks fine to me - what's the issue here?

Reply:
Oh nvm, you fixed it